### DISCUSSION TITLE
【求助】有邦布殡葬服务吗？

### DISCUSSION THREAD

**Discussion by KSN-tt41** [share121/inter-knot#609] [其他]
（虚构史学家预警）
我的一位朋友以太抗性是绿色，按理说是不被允许进入空洞的，但这家伙一个月前不知道从哪搞了个萝卜，说要找什么黄金城，带我一夜暴富之类的，到时候再拿着这些钱当启动资金，去光映广场随便租个店面卖点小吃，舒舒服服过一辈子。

如题，自他进入空洞的那天起就再没有任何消息，这一个月内我找过了各种方法，治安局、空洞调查协会我都跑过了，他们都说要我珍惜当下，不要一味地沉浸在过去，说我不值得为一个利欲熏心抛下朋友的家伙缅怀，说什么我的未来还很长之类的...之后我也砸了不少丁尼在绳网上，希望能有绳匠可以帮忙，但大多最后都不了了之，几位善良的绳匠也让我不要有太大的期待，会尽量为我找回朋友的遗物。我也看开了，拜托对方如果在委托地点附近看到以骸就上报给空洞调查协会吧，这或许对他来说是最好的结果了。在此先向之前一直帮助我的绳匠们致谢。

但还有一件事我一直耿耿于怀，作为他的好朋友之一我想我至少要给他举办一场葬礼，但又想不出该怎么办。我的那位朋友生前最喜欢的就是邦布，迫不得已来到绳网上发布委托，希望可以有热心的绳匠来完成我这一个自私的愿望。以下是我的一些碎碎念，和委托关系不大，如果有绳匠在因为帮一个贪财的亡命徒举办葬礼而感到膈应，也可以看看下面以我主观视角叙述的他的经历。

我可以为我这位朋友的人格做担保，他和其他闯入黄金城的家伙绝对不一样，他原本用来作启动资金的钱被他的父母偷去给弟弟买房了，我不懂法，也没办法帮他什么...法庭上他的父母请律师作了伪证，非常感谢愿意低价帮助我们的律师，他在法庭上据理力争，但终究还是不敌对方，被打入了逻辑闭环，除非我们也做伪证，否则我们真的一点胜算都没有了。最后朋友还是不愿意，拒绝了律师先生的好意，赔偿了父母一大笔钱，他也因此背上了巨额的债务。如果不去黄金城，他也早晚有一天会被追债的人打死，但变成以骸增加空洞危险程度什么的，我也先替这位朋友道个歉，对不起各位绳匠和调查员们，是我们考虑不周了，这是我们的最后一根救命稻草，我们实在是没有任何其他办法了。

最后，感谢愿意读到这里的绳匠们，愿你们今后不会碰到类似的情况，祝身体健康，万事如意，也祝全天下的家庭美满和睦，财运亨通。

**Comment by Hongxuli2006** [comment]
已阅

**Comment by KSN-tt41** [comment]
哇，什么时候发的，我号这是让人盗了？兄弟们别信这是虚构史学家卖惨搞非法募捐的，空洞哪是想进就进想去哪就去哪的，一看就不是本地人没有空洞进出经验，举办了

**Comment by Pai-pi** [comment]
这个委托我接了

**Comment by KSN-tt41** [comment]
这个回复语气，难道您是那位传奇新人绳匠？？？绳匠老师能给个赛博签名吗我是你的粉丝啊（飞扑）

**Comment by GanlinSama** [comment]
本来还想问问楼主是不是熊希人，因为那天下去的时候看见有个被困住的熊希人，结果是构史学家啊……

**Comment by KSN-tt41** [comment]
楼主是纯种人希人的说
